pkg/virt-api/rest/subresource.go

```diff
@@ -72,6 +80,25 @@ const (
 type validation func(*v1.VirtualMachineInstance) error
 type URLResolver func(*v1.VirtualMachineInstance, kubecli.VirtHandlerConn) (string, error)
 
+type Informers struct {
+	PodInformer cache.SharedIndexInformer
+}
+
+func NewInformer() *Informers {
```

Comment:
I would get rid of this completely and just use the the factory in the other file above directly.